externals/kyuubi-spark-sql-engine/src/main/resources/META-INF/NOTICE

```diff
@@ -23,269 +23,5 @@ Copyright 2019 The Apache Software Foundation
 Apache Thrift
 Copyright 2006-2010 The Apache Software Foundation.
 
-Google Guava Version 18.0
-* Copyright (C) 2009 The Guava Authors
```

Comment:
Accidentally deleted? Seems Guava still be include.

Reply:
The recent version Guava does not ship NOTICE in the jar

Reply:
Get, LGTM, I will have a test locally.